src/App/App.tsx

```diff
@@ -86,7 +86,7 @@ export default class App extends Component<{}, AppStateInterface> {
       this.handleLogout();
     } else {
       this.setState({
-        user: { username, token },
```

Comment:
Why removing the token from the App state?

Reply:
I think might be a mistake. Let's wait @tmkn reply.

Reply:
Because the `token` itself is not defined in the `AppStateInterface` (I didn't change the interface)
```ts
export interface AppStateInterface {
  error?: FormError;
  logoUrl: string;
  user: {
    username?: string;
  };
  scope: string;
  showLoginModal: boolean;
  isUserLoggedIn: boolean;
  packages: [];
  isLoading: boolean;
}
```
only `username` is

Reply:
Good point, I think should be defined there. You just found a bug :-)

Reply:
Yeah, good catch @tmkn 😄

Reply:
As far as I can see, the `token` is only ever written to the App state, never read from, it's written and read from, from the `storage` object. instead. (`storage.getItem('token')` etc)

Also only 1 test failed(which was just testing the presence of the token), so I assumed that ts is right and the token doesn't belong into the App state and removed the assignments.

But I can add them back of course

Reply:
Good find.
I also found it missing and added https://github.com/verdaccio/ui/pull/85/files#diff-2d39dbde05630bdfe781f3a465e7857bR28

Reply:
You are completely right @tmkn 🚀 . It is useless. We persist with local storage. Go ahead then.

Reply:
Good catch @griffithtp !